sputnikdao2/src/upgrade.rs

```diff
@@ -1,14 +1,16 @@
 //! Logic to upgrade Sputnik contracts.
 
+use near_sdk::serde_json::json;
 use near_sdk::Gas;
 
 use crate::*;
 
 const FACTORY_KEY: &[u8; 7] = b"FACTORY";
-const DEFAULT_FACTORY_ID: &str = "sputnik-dao.near";
+const DEFAULT_FACTORY_ID: &str = "factory";
```

Comment:
we need to find a way to tell if we are on localnet/testnet/mainnet here

Reply:
Good idea - probably worth a separate issue